src/main/java/db/migration/V10__form_session_add_app_id.java

```diff
@@ -0,0 +1,15 @@
+package db.migration;
+
+import org.flywaydb.core.api.migration.spring.SpringJdbcMigration;
+import org.springframework.jdbc.core.JdbcTemplate;
+
+/**
+ * Add appId to form session
+ */
+public class V10__form_session_add_app_id implements SpringJdbcMigration {
+    @Override
+    public void migrate(JdbcTemplate jdbcTemplate) throws Exception {
+        jdbcTemplate.execute("ALTER TABLE formplayer_sessions " +
+                "ADD appid VARCHAR DEFAULT '1'");
```

Comment:
should the default not by NULL so we don't get confused if there was an actual app associated with the form?